src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -285,6 +285,24 @@ public String toString() {
 
   };
 
+  public static final Builder<Tuple> TUPLE = new Builder<Tuple>() {
```

Comment:
@z0mg can you please add a comment about the different between TUPLE_ZSET and TUPLE?
Or perhaps we better even rename this builder?

Reply:
@gkorland ZPOPMAX and ZPOPMIN without count returns only one Tuple. Returning a Tuple should be better instead of returning a Set of single Tuple. This builder helps on that purpose.